stephanie-smith/short-response.md

```diff
@@ -0,0 +1,66 @@
+# Problem Set 2.2 - Variables and Control Flow
+## Short Response Questions
+
+1. **What does the code below log? Why?**
+   ```javascript
+    {
+      let singer = 'Omar Apollo';
+    }
+
+    console.log(`My favorite singer is ${singer}`);
+   ```
+   - It logs "My favorite singer is Omar Apollo".
+
+2. **What happens when we run the following code? Why?**
+   ```javascript
+   const favorite = 'Juan Pablo';
+   console.log(`Our favorite Marcy Lab family member is ${favorite}!`);
+
+   favorite = 'Maya';
+   console.log(`Actually, ${favorite} is my favorite.`);
+   ```
+   - When you run the code, you get an error about the type used for the variable `favorite`, which is a constant. This means they cannot be redefined.
```

Comment:
We didn't re_define_ the variable `favorite`, we **reassigned** it. There's a big difference between the two. Make this change.